models/balancer/ethereum/balancer_v2_ethereum_lbps.sql

```diff
@@ -0,0 +1,118 @@
+{{
+    config(
+        schema = 'balancer_v2_ethereum',
+        alias='lbps',
+        unique_key = ['pool_id'],
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "balancer_v2",
+                                    \'["stefenon"]\') }}'
+    )
+}}
+
+{% set create_lbp_contracts = ['LiquidityBootstrappingPoolFactory_call_create',
+                                'NoProtocolFeeLiquidityBootstrappingPoolFactory_call_create'] %}
+{% set lbp_weight_upgrade_contracts = ['LiquidityBootstrappingPool_evt_GradualWeightUpdateScheduled',
+                                        'NoProtocolFeeLiquidityBootstrappingPool_evt_GradualWeightUpdateScheduled'] %}
+{% set non_lbp_tokens = ('0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+                        '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+                        '0x6b175474e89094c44da98b954eedeac495271d0f',
+                        '0x88acdd2a6425c3faae4bc9650fd7e27e0bebb7ab',
+                        '0x64aa3364f17a4d01c6f1751fd97c2bd3d7e7f1d5'
+                        ) %} -- WETH, USDC, DAI, MIST & OHM
+
+WITH lbps_call_create AS (
+        {% for create_lbp_contract in create_lbp_contracts %}
+        SELECT * FROM {{ source('balancer_v2_ethereum', create_lbp_contract) }}
+        {% if not loop.last %}
+        UNION ALL
```

Comment:
Hi @Stefenon , you may rewrite the jinja loop in the following way:
```suggestion
{% set create_lbp_contracts = [
        source('balancer_v2_ethereum', 'LiquidityBootstrappingPoolFactory_call_create'),
        source('balancer_v2_ethereum', 'NoProtocolFeeLiquidityBootstrappingPoolFactory_call_create')
] %}
{% set lbp_weight_upgrade_contracts = [
        source('balancer_v2_ethereum', 'LiquidityBootstrappingPool_evt_GradualWeightUpdateScheduled'),
        source('balancer_v2_ethereum', 'NoProtocolFeeLiquidityBootstrappingPool_evt_GradualWeightUpdateScheduled')
] %}
{% set non_lbp_tokens = ('0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
                        '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
                        '0x6b175474e89094c44da98b954eedeac495271d0f',
                        '0x88acdd2a6425c3faae4bc9650fd7e27e0bebb7ab',
                        '0x64aa3364f17a4d01c6f1751fd97c2bd3d7e7f1d5'
                        ) %} -- WETH, USDC, DAI, MIST & OHM

WITH lbps_call_create AS (
        {% for create_lbp_contract in create_lbp_contracts %}
        SELECT * FROM {{ create_lbp_contract }}
        {% if not loop.last %}
        UNION ALL
```